src/site/layouts/basic_landing_page.drupal.liquid

```diff
@@ -50,6 +50,65 @@
             {% include "src/site/includes/support_resources_search_bar.drupal.liquid" %}
           </div>
 
+          <div class="usa-alert usa-alert-info" role="alert">
+            <div class="usa-alert-body">
+                <h3 class="usa-alert-heading">
+                    We're currently in beta testing
+                </h3>
+                <div class="processed-content">
+                  <div itemprop="text">
+                    <p>Welcome to resources and support, a new part of <a href="/">VA.gov</a>. We'll be adding more articles and topics in the weeks and months ahead, so please check back often.</p>
+                  </div>
+                </div>
+            </div>
+          </div>
+
+          {% comment %}
+            On hold until we have a large set of audience-types
+          {% endcomment %}
+          {% if buildtype != 'vagovprod' %}
```

Comment:
Feature flag added here

Reply:
Staging.va.gov could wind up w/broken links but we allow those in environments less than prod anyway